### PR TITLE
tkt-72067: Uppercase the RELEASE specified during create

### PR DESCRIPTION
**Pull request by skarekrow**
This helps avoid lazy sysadmin syndrome

Reported by: Michael W Lucas

FreeNAS Ticket: #72067

### CODE REVIEW

File: iocage_lib/iocage.py [hunk 1]
```diff
@@ -550,9 +550,14 @@ def create(self,
                 _callback=self.callback,
                 silent=self.silent)
 
+        if os.path.isdir(
+            f'{self.iocroot}/releases/{release.upper()}'
```
Comment (sonicaj) [inline iocage_lib/iocage.py]: Should we perhaps to be on the safe side do something like `(release or '').upper()` for someone who might be using lib directly ?

**Review by sonicaj — DISMISSED** [review]
LGTM, just a minor note

File: iocage_lib/iocage.py (hunk 1 above)
Comment (skarekrow) [reply iocage_lib/iocage.py]: That would trigger a false positive as `f'{self.iocroot}/releases/` almost always exists

Comment (sonicaj) [reply iocage_lib/iocage.py]: that is true, i missed that - maybe in this case we should just check release is true

**Review by sonicaj — APPROVED** [review]

**Comment by sonicaj** [comment]
Thank you ;)

**Pull request merged**